ci/unit_tests/workspace_delete/main_test.py

```diff
@@ -71,6 +71,7 @@ def _deleteAllWorkspaces(self):
 
         assert len(workspaces) == 0
 
+    @pytest.mark.skipif(os.environ.get('TRAVIS_EVENT_TYPE') != "cron", reason="This test is nightly build only.")
     @pytest.mark.parametrize('envVarNameUsername, envVarNamePassword', [('WA_USERNAME', 'WA_PASSWORD')])
     def test_deleteById(self, envVarNameUsername, envVarNamePassword):
```

Comment:
I think that this test should be enabled for non-nightly builds

Reply:
Yes, I forgot to save changes :)